leshan-integration-tests/credentials/client.csr

```diff
@@ -1,9 +1,9 @@
------BEGIN NEW CERTIFICATE REQUEST-----
+-----BEGIN NEW CERTIFICATE REQUEST-----

```

Comment:
It seems the end of line has changed from unix style to windows style. unix style should be used.